include/ruby/internal/intern/class.h

```diff
@@ -88,8 +88,8 @@ VALUE rb_define_class_id(ID id, VALUE super);
  * @post        `outer::id` refers the returned class.
  * @note        If a class named `id` is  already defined and its superclass is
  *              `super`, the function just returns the defined class.
- * @note        The  compaction  GC does  not  move  classes returned  by  this
- *              function.
+ * @note        The GC does not collect nor move move classes returned by this
```

Comment:
There's a duplicated "move" everywhere.

```suggestion
 * @note        The GC does not collect nor move classes returned by this
```

Reply:
🤦 .

Corrected.

Reply:
I don't see it corrected in this file.

Reply:
Yeah, I'm an idiot, should be good now.